helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/ReadOnlyWagedRebalancer.java

```diff
@@ -0,0 +1,88 @@
+package org.apache.helix.controller.rebalancer.waged;
```

Comment:
Let's put this in the util or tool package? It should not be used as a rebalancer.